sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -33,13 +33,17 @@ import kalix.springsdk.testmodels.action.ActionsTestModels.PutWithoutParam
 import kalix.springsdk.testmodels.action.ActionsTestModels.StreamInAction
 import kalix.springsdk.testmodels.action.ActionsTestModels.StreamInOutAction
 import kalix.springsdk.testmodels.action.ActionsTestModels.StreamOutAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.PublishToTopicAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.RestAnnotatedSubscribeToValueEntityAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.RestWithPublishToTopicAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeToTopicAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeToTwoTopicsAction
-import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeToValueEntityAction
-
+import kalix.springsdk.testmodels.subscriptions.PubSubTestModels
+import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.{
+  PublishToTopicAction,
+  RestAnnotatedSubscribeToEventSourcedEntityAction,
+  RestAnnotatedSubscribeToValueEntityAction,
+  RestWithPublishToTopicAction,
+  SubscribeToEventSourcedEntityAction,
+  SubscribeToTopicAction,
+  SubscribeToTwoTopicsAction,
+  SubscribeToValueEntityAction
+}
```

Comment:
Does that comes from IntelliJ or scalafmt? 
And if from Intellij, isn't scalafmt reverting it?

Reply:
again?? darn! I change it a couple of times back but it seems IntelliJ has beaten me

Reply:
let me check who is the culprit here. Apart from me.

Reply:
It was Intellij. Disabled it 
<img width="397" alt="Screenshot 2022-09-09 at 13 53 02" src="https://user-images.githubusercontent.com/1381621/189334564-fbdff317-a972-45a1-bd01-fa9b4532a8ee.png">